lib/node-repo.js

```diff
@@ -25,41 +25,17 @@ function updatePrWithLabels (options, labels) {
     return
   }
 
-  fetchExistingLabels(options, (err, existingLabels) => {
```

Comment:
We shouldn't need to fetch existing labels when adding PR labels, cause the updated github dependency has a new method for *adding labels*, rather than editing the whole issue like we did before.